### PR TITLE
fix: Order Page and Completed Order Page: Change "Go to event" 

### LINKED ISSUE #5616: Order Page and Completed Order Page: Change "Go to event"
Event payment page and completed order page have a link "Go to event". Please change it as follows:

1. On event payment page, do not show "Go to event" anymore and delete the area where the links wha located. Basically, do not show an empty space there.

2. On the completed order page enhance it as follows:
    * add a website icon in front of text
    * Change "Go to event" to "Go to event page"

Order Page

Screenshot from 2020-11-13 22-17-45

Completed Order Page

Screenshot from 2020-11-13 22-29-04

### PR DESCRIPTION
**Pull request by daretobedifferent18**
Fixes #5616 

#### Short description of what this resolves:
Enhance the Order and complete order page

#### Changes proposed in this pull request:

- add a param for compeleted order so that if n only if order is completed then only 'go to event page' link is visible
- add website icon

#### ScreenShots

Order Page
-
Screenshot from 2020-11-14 04-56-02
Complete Order Page
-
Screenshot from 2020-11-14 04-55-18

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/templates/components/orders/event-info.hbs [hunk 1]
```diff
@@ -22,7 +22,10 @@
...
   </div>
-  <div class="ui padded segment">
-    <a href="{{this.data.event.url}}">{{t 'Go to event'}}</a>
-  </div>
+  {{#if this.completed}}
+    <div class="ui padded segment">
+      <i class="globe icon"></i>
+      <a href="{{this.data.event.url}}">{{t 'Go to event page'}}</a>
+    </div>
+  {{/if}}
```
Comment (daretobedifferent18) [inline app/templates/components/orders/event-info.hbs:30]: Changed this

File: app/templates/orders/view.hbs [hunk 2]
```diff
@@ -19,32 +19,42 @@
         @data={{this.model.order}}
         @event={{this.model.event}}
         @eventCurrency={{this.model.order.event.paymentCurrency}} />
-      <br/>
+    </div>
+    <div class="mobile hidden six wide column">
+      <Orders::EventInfo
+        @data={{this.model.order}} 
+        @completed={{true}} />
```
Comment (daretobedifferent18) [inline app/templates/orders/view.hbs:26]: and this only rest are due to merge conflict

**Review by sachinchauhan2889 — CHANGES_REQUESTED** [review]
i think you forgot about first point. please remove empty spaces in between boxes
all other things working fine.

review

**Comment by sachinchauhan2889** [comment]
you can refer my previous commits for removing empty spaces.
https://github.com/fossasia/open-event-frontend/pull/5561
https://github.com/fossasia/open-event-frontend/pull/5464

**Comment by daretobedifferent18** [comment]
> you can refer my previous commits for removing empty spaces.
> #5561
> #5464

I think that is already done in ur PR's but I don't know why is this not updated the code here

**Comment by sachinchauhan2889** [comment]
> I think that is already done in ur PR's but I don't know why is this not updated the code here

i have remove empty space in pr #5618. you can skip this. otherwise this create conflict.

**Comment by daretobedifferent18** [comment]
> > I think that is already done in ur PR's but I don't know why is this not updated the code here
> 
> i have remove empty space in pr #5618. you can skip this. otherwise this create conflict.

so nothing to do anything now?

**Comment by daretobedifferent18** [comment]
Due to some error Ill be creating new PR

**Comment by daretobedifferent18** [comment]
@iamareebjamal PLease see the changes I have made

File: app/templates/orders/view.hbs [hunk 3]
```diff
@@ -28,7 +28,8 @@
     </div>
     <div class="six wide column">
       <Orders::EventInfo
-        @data={{this.model.order}} />
+        @data={{this.model.order}}
+        @completed={{true}} />
```
Comment (iamareebjamal) [inline app/templates/orders/view.hbs:32]: Why

Comment (daretobedifferent18) [reply app/templates/orders/view.hbs:32]: bcz acc to the enhacement, we want 'go to event page' only for complete order page

Comment (daretobedifferent18) [reply app/templates/orders/view.hbs:32]: @iamareebjamal Please let me know

Comment (iamareebjamal) [reply app/templates/orders/view.hbs:32]: You don't need this, you already have order as an argument

**Comment by daretobedifferent18** [comment]
@iamareebjamal  Done

**Review by iamareebjamal — APPROVED** [review]

**Pull request merged**